internal/cephfs/nodeserver.go

```diff
@@ -311,18 +311,11 @@ func (ns *NodeServer) mount(
 
 	log.DebugLog(ctx, "cephfs: mounting volume %s with %s", volID, mnt.Name())
 
-	var mountOptions []string
-	if m := volCap.GetMount(); m != nil {
-		mountOptions = m.GetMountFlags()
-	}
+	err = ns.getMountOptions(mnt, volOptions, volCap)
```

Comment:
I think setMountOptions is better suited. 
Please include the below "ro" detection in the function too.

Reply:
Done.